subsys/bluetooth/mesh/Kconfig

```diff
@@ -1666,4 +1666,12 @@ config BT_MESH_DEBUG_USE_ID_ADDR
 
 endif # BT_MESH_LOG_LEVEL_DBG
 
+config BT_MESH_STATISTIC
+	bool "The frame handling statistics [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  Module gathers statistics of received, relayed, and transmitted
+	  frames. This help to estimate the quality of communication and
+	  the sufficiency of configured advertiser instances and buffers.
```

Comment:
I'm uncertain that this helps to estimate sufficiency of buffers as messages are dropped due to insufficient buffers if `bt_mesh_adv_create` function fails, but this is not covered in this module.